src/Symfony/Component/Profiler/DataCollector/RuntimeDataCollectorInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Profiler\DataCollector;
+
+use Symfony\Component\Profiler\ProfileData\ProfileDataInterface;
+
+/**
+ * RuntimeDataCollectorInterface.
+ *
+ * @author Jelte Steijaert <jelte@khepri.be>
+ */
+interface RuntimeDataCollectorInterface
+{
+    /**
+     * Collects data when profiler is triggered.
+     *
+     * @return ProfileDataInterface
+     */
+    public function collect();
```

Comment:
I think the name of this method is not quite appropriate. The method does not actually collect the data (the data is collected during the run of the application), but only return the collected data. Hence I think `getCollectedData()` would be more appropriate.